File: frontend/src/types/venueMap.ts
export interface VenueMapPoint {
  id: number;
  venue_id: number;
  label: string;
  point_type: string;
  x: number;
  y: number;
}

export interface VenueMapPointCreate {
  venue_id: number;
  label: string;
  point_type: string;
  x: number;
  y: number;
}

File: backend/app/schemas/venue_map.py
from pydantic import BaseModel, Field

class VenueMapPointCreate(BaseModel):
    venue_id: int
    label: str
    point_type: str
    x: float = Field(ge=0, le=100)
    y: float = Field(ge=0, le=100)

class VenueMapPointResponse(BaseModel):
    id: int
    venue_id: int
    label: str
    point_type: str
    x: float
    y: float

    class Config:
        from_attributes = True